### DISCUSSION TITLE
Information on crystal size/vendor

### DISCUSSION THREAD

**Discussion by drewblank** [OpenGammaProject/Open-Gamma-Detector#40] [Q&A] [answered]
What crystal size works best with this setup? Also any recommended vendors? Also how does the crystal size relate to the photomultiplier area?

**Comment by NuclearPhoenixx** [comment, marked as answer]
Hi there,

[hardware#scintillator-assembly](https://github.com/OpenGammaProject/Open-Gamma-Detector/tree/main/hardware#scintillator-assembly) covers most of the basics.

In short: 1x1 Inch NaI(Tl) scintillators coupled with the 2x2 SiPM array works the best, generally everything >= maybe 1x1 inch it's highly recommended you use the 2x2 array to get the best energy resolution. I got multiple crystals now from OST Photonics, they seem to be pretty reliable and have great prices.

Hope that answers your questions.
